portfolio/src/main/webapp/index.html

```diff
@@ -6,48 +6,107 @@
+    -->
   <body>
     <div id="content">
-      <h1>Lisandro's Portfolio</h1>
-      <div id="intro">
-        <div id = "welcome">
-            <img src="/images/posse.jpg">
-            <p>This is my portfolio.</p>
-            <p> This page will serve as a repository of my relevant work in Computer Science and Education as well as a some fun things I do on the side! </p>
-            <button onclick="swapHidden('welcome','intro-main')">Learn More</button>
-        </div>
-        <div id="intro-main">
-            <h1>About Me</h1>
-            <p>Hello! my name is Lisandro Mayancela and I am currently pursing a Bachelor of Science in 
+      <div id="header">
+        <h1 class="header-text">Lisandro Mayancela</h1>
+      </div>
+      <div id="main">
+        <!--Below is the code for the introduction page.-->
+        <div id="intro">
+          <div id="main-content">
+            <div id = "welcome">
+              <img src="/images/main.jpg">
+                <h2>Welcome to my portfolio!</h2>
+                <p>This page will serve as a repository of my relevant work in Computer Science and Education as well as a some fun things I do on the side!</p>
+                <button onclick="swapHidden('welcome','intro-main')">Learn More</button>
+            </div>
+            <div id="intro-main">
+              <img src="/images/posse.jpg">
+              <h1>About Me</h1>
+              <p>Hello! my name is Lisandro Mayancela and I am currently pursing a Bachelor of Science in 
                 Computer Science as well as a minor in East Asian Studies and Education Studies.
                 My interest in Computer Science actually began at a young age when I fell in love with gaming and
                 wanted to help make games so that I can bring these experiences to the next generation of kids.
                 As I advanced through high school and learned more about CS, my goals shifted to wanting to work on
                 Web/App development. In college, this would once again evolve as I now have a passion for eventually
                 bringing my skills into education so that I can help foster the next generation of PoC students in tech
                 so that the field can be diversified and all students can have an equal opportunity to get involved in CS.</p>
-            <p>Click on the buttons below to learn more about myself and the work I have done!</p>
-            <div id="buttons">
+              <p>Click on the buttons below to learn more about myself and the work I have done!</p>
+              <div class="buttons">
                 <button onclick="swapHidden('intro-main','projects')">Projects</button>
                 <button onclick="swapHidden('intro-main','gallery')">Gallery</button>
                 <button onclick="swapHidden('intro-main','interests')">Interests & Hobbies</button>
+              </div>
+            </div>
+            <!--Below is the code for the projects page.-->
```

Comment:
This comments are great! Makes it very easy to follow the structure of your page.